website/templates/emails/confirm_preprints_branded.txt.mako

```diff
@@ -1,6 +1,10 @@
 Hello ${user.fullname},
 
+% if branded_preprints_provider == "Thesis Commons":
```

Comment:
A more robust check might be:


```mako
% if preprint_word != 'preprint':
```

Reply:
Yes, or this could also use the plural word instead or "Preprints" on L3/6

Reply:
Discussed with Nici on FD, the special casing can be remove here too:

> Welcome to SoxArxiv, powered by the Open Science Farmework.

> Welcome to Thesis Commons, powered by the Open Science Framework.